src/main/resources/view/BookListCard.fxml

```diff
@@ -28,10 +28,14 @@
         <Label fx:id="name" text="\$first" styleClass="cell_big_label" wrapText="true" />
       </HBox>
       <FlowPane fx:id="bookCategoryTags" />
+      <!-- Ensures that the author line will not be rendered if the book doesn't have an author-->
       <Label fx:id="authors" styleClass="cell_small_label" visible="false" managed="false" wrapText="true" />
       <Label fx:id="isbn" styleClass="cell_small_label" text="\$isbn" />
       <FlowPane fx:id="bookAvailableTag" />
       <FlowPane fx:id="bookBorrowTag" />
+      <!-- Ensures that the borrower and book return date line will not be rendered if the book is not borrowed-->
+      <Label fx:id="borrower" styleClass="cell_small_label" visible="false" managed="false" wrapText="true" />
```

Comment:
Good that this label is set to book by default!

If we assume Book is ummutable, and the actual object is always being replaced instead of modified, then this will be correct.

Reply:
Just in case, I will also set the visibility of the borrower and return date in BookCard.java to false if the book is available.
```
        if (book.isAvailable()) {
            borrower.setVisible(false);
            borrower.setManaged(false);
            bookReturnDate.setVisible(false);
            bookReturnDate.setManaged(false);
```